.github/workflows/static-analysis.yml

```diff
@@ -47,6 +47,8 @@ jobs:
 
     name: Facade DocBlocks
 
+    if: ${{ github.repository_owner == 'laravel' && (github.ref_name == 'master' || github.ref_name == '9.x') }}
```

Comment:
I wish I could have used a regex here to update the branch checking to `*.x` but I'm unsure how to do this in an if statement: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif